tests/test_basic.py

```diff
@@ -70,6 +73,7 @@ def test_show_existing(tmpdir, runner, create):
         'test.ics',
         'SUMMARY:harhar\n'
         'DESCRIPTION:Lots of text. Yum!\n'
+        'PRIORITY:0\n'
```

Comment:
Apparently GitHub swallowed my comment: Adding this line to all tests shouldn't be necessary, todoman should use `0` as value if `PRIORITY` is not defined.